beebop/versions.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from beebop import __version__
+
+
+def get_version(components):
```

Comment:
What is the use-case for making components configurable?  This will be easier to write and maintain with no args I think. Then drop the loop and assemble directly